src/app/components/logo-title/logo-title.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="logo-title-component">
+  <a
+    href="/all"
+    style="display: flex; width: 100%; justify-content: space-between"
```

Comment:
It is not a good approach to use styles like this. Please **`Do not use inline styles`**.